### ISSUE TITLE
Method chaining

### ISSUE THREAD

**Issue by DevDavido** [mexitek/phpColors#40]
Thanks for this nice library! One thing that I constantly encounter is that I can't perform multiple actions (lighten, mix, ...) on one color but instead need to recreate Color objects for each step. I would suggest to return $this instead self::hslToHex()? This would break compatibility, so it would need an major version jump? Would go well though with @finwe's pull request #35.

**Comment by DevDavido** [comment]
Would you merge such a pull request @mexitek?

**Comment by mexitek** [comment]
Yes I would. Feel free to contribute. We can version bump if necessary.
